tika-parsers/src/test/resources/test-properties/TesseractOCRConfig-partial.properties

```diff
@@ -15,4 +15,10 @@
 
 language=fra+deu
 timeout=240
-minFileSizeToOcr=1
\ No newline at end of file
+minFileSizeToOcr=1
+
+enableProcessing=0
```

Comment:
yep resolved in the main properties file, @Zarana-Parekh but not this test one please update this too.